src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string FrameworkName = ".NET Core";
+        private static Architecture? s_processArch = null;
+        private static readonly bool s_is64BitProcess = IntPtr.Size == 8;
+
+        public static string FrameworkDescription
+        {
+            get
+            {
+                return FrameworkName;
+            }
+        }
+
+        public static Architecture ProcessArchitecture
+        {
+            get
+            {
+                if (null == s_processArch)
+                {
+                    if (Architecture.Arm == OSArchitecture)
```

Comment:
I think I understand what you are inferring. When Arm64 is added this will return true for 64 bit process, but when arm64 is added, i will change this implementation to depend on whatever api support windows provides. Right now there is not clarity on Arm64 and how that will work in wow environment if at all in windows. 

And here i am trying to get the running process architecture not the processor hardware architecture.